xarray/tests/test_units.py

```diff
@@ -0,0 +1,1515 @@
+import operator
+from distutils.version import LooseVersion
+
+import numpy as np
+import pytest
+
+import xarray as xr
+from xarray.core.npcompat import IS_NEP18_ACTIVE
+
+pint = pytest.importorskip("pint")
+DimensionalityError = pint.errors.DimensionalityError
+
+
+pytestmark = [
+    pytest.mark.skipif(
+        not IS_NEP18_ACTIVE, reason="NUMPY_EXPERIMENTAL_ARRAY_FUNCTION is not enabled"
+    ),
+    # pytest.mark.filterwarnings("ignore:::pint[.*]"),
+]
```

Comment:
This is unused at the moment. If NEP18 is off, you should skip the whole test suite with a line right after import_or_skip("pint").

Reply:
Ah sorry, my bad. I didn't know that ``pytestmark`` is a special pytest keyword.